### PR TITLE
Got rid of entries with error messages. 

### PR DESCRIPTION
**Pull request by inescgu**

### CODE REVIEW

File: all.bib [hunk 1]
```diff
@@ -6011,6 +6011,220 @@ @inproceedings{gyongyi2004combating
...
+@article{white2009exploratory,
+  author = {Ryen W White and Resa A Roth},
+  journal = {Synthesis lectures on information concepts, retrieval, and services},
+  number = {1},
+  pages = {1--98},
+  title = {Exploratory search: Beyond the query-response paradigm},
+  volume = {1},
+  year = {2009},
+}
+@article{kelly2009methods,
```
Comment (percyliang) [inline all.bib:6219]: Why is this added, but removed from the rb file?

Comment (inescgu) [reply all.bib:6219]: In trying to fix my errors, I accidentally duplicated all of my entries. I went ahead and removed the duplicates - although I see they're not visible on the pull request.

Comment (inescgu) [reply all.bib:6219]: I'm not entirely sure either what's happening, because all the entries are showing up in my local copy of the .rb file and have compiled in the all.bib file, but are not showing up on the pull request.

**Review by percyliang — COMMENTED** [review]
I removed `data/inescgu.rb` from the master branch for now since it was causing errors...

I'm not sure why your PR removes all the entries from `data/inescgu.rb`?  Can you close this PR and create another PR that adds that file and the necessary entries?  You should run `make` to make sure there are no errors.

The general strategy is not to use fields like editor, vlume, publisher, etc., and just use `inproceedings` and `article` whenever possible.  Take a look at the other files to get a sense of the norms.

**Comment by inescgu** [comment]
Got it, thank you.
